model/src/main/java/org/projectnessie/api/TreeApi.java

```diff
@@ -126,8 +126,11 @@ public EntriesResponse getEntries(
       @Pattern(regexp = Validation.REF_NAME_OR_HASH_REGEX, message = Validation.REF_NAME_OR_HASH_MESSAGE)
       @Parameter(description = "name of ref to fetch from")
       @PathParam("ref")
-          String refName)
-          throws NessieNotFoundException;
+          String refName,
+      @Parameter(description = "list of value types to return. Return all if empty")
```

Comment:
It feels like a more generic `filter` query parameter might allow for extensibility

Reply:
How do you mean? Just rename the parameter or some other object type?

Reply:
some object type

Reply:
Sorry @laurentgo, its still not clear to me what you are asking for.